File: go/excel_column.go
package main

import "fmt"

func solution(n int) string {
    var a []byte
    if n == 0 {
        return ""
    }
    for n > 0 {
        mod := byte(n % 26)
        if mod == 0 {
            a = append([]byte{90}, a...)
            n = int(n / 26) - 1
        } else {
            a = append([]byte{mod + byte(64)}, a...)
            n /= 26
        }
    }
    return string(a)
}

func main(){
    fmt.Println(solution(0))
    fmt.Println(solution(14))
    fmt.Println(solution(134))
}

File: python/hexlet/excel_column.py
def solution(n):
    a = []
    if n == 0:
        return ""
    while n > 0:
        mod = n % 26
        if mod == 0:
            a.append("Z")
            n = n // 26 - 1
        else:
            a.append(chr(mod + 64))
            n = n // 26

    return "".join(a)[::-1]

if __name__ == '__main__':
    print(solution(0))
    print(solution(14))
    print(solution(134))
